R/adjust_speed.R

```diff
@@ -0,0 +1,56 @@
+
+#' @title Estimate cumtime based on global speed
+#'
+#' @description Estimate cumtime of NA values of a GPS data based on the average
+#' speed of the valid GPS rows.
+#'
+#' @param gps_data A data.table with GPS-like rows.
+#' @param min_speed Minimum speed to be considered as valid. Values below minimum
+#' speed will be estimated as if they were NA. Default value is 2km/h.
+#' @param max_speed Maximum speed to be considered as valid. Values above maximum
+#' speed will be estimated as if they were NA. Default value is 80km/h.
+#' @param new_speed Speed to replace missing values as well as values outside
+#' min_speed and max_speed range. The default value is the average speed of the
+#' gps data.
+#' @details This function estimates speed values using the average speed of the
+#' whole GPS data, ignoring infinite, NA, and NAN values. Values above 80km/h and
+#' below 2km/h are also ignored and their values will be updated with the average.
```

Comment:
the `@details` section is repeating the default behavior of the `new_speed` parameter. Perhaps we reduce redundancy in the documentation by dropping the  `@details`  section